### ISSUE TITLE
Workouts not syncing from PC to mobile devices

### ISSUE THREAD

**Issue by mFabio98** [zoffline/zwift-offline#194]
I like to use Zwift from my Android phone, however I cannot when I have a workout scheduled since it does not sync from my PC to the phone.
Here's what I normally do:
- I create/import the workout on intervals.icu
- I export it from intervals.icu in .zwo format
- I place it in *user*\Documents\Zwift\Workouts\*player_ID*
- I start zwift and find my workout in Zwift under 'Custom Workouts'
Normally, after opening Zwift on my PC, it should sync via cloud to my phone, but with zwift-offline it does not. 
It would be even better if I could simply upload the workout to a folder on the home server where I have zwift-offline installed and the workout gets delivered to all my devices. 
It's not properly an issue but a lack of a feature.
Anyway, phenomenal work.
Thanks.

**Comment by oldnapalm** [comment]
Yes, it's a missing feature, `/api/zfiles` is not implemented.

https://github.com/zoffline/zwift-offline/blob/001e3aaa45db42a19ca4bd567feaa94ad25e9986/zwift_offline.py#L1246

https://github.com/zoffline/zwift-offline/blob/001e3aaa45db42a19ca4bd567feaa94ad25e9986/zwift_offline.py#L1281

**Comment by mFabio98** [comment]
Thank you @oldnapalm.
I'll keep a look at this, unfortunately I can't help, I'm definetly not skilled enough.

**Comment by oldnapalm** [comment]
Added initial zfiles support in https://github.com/zoffline/zwift-offline/tree/zfiles

Needs testing, only tried add/remove/download on Windows.

**Comment by mFabio98** [comment]
I just tested it and I'm glad to report that **it also works for Android devices**, I was able to sync all the custom workouts from my PC to my phone. 
I executed from command line instead of having it as a service in order to see the verbose output and I couldn't see any error message. 
I haven't tested removing the workouts though, I'll get back in touch when I'll do it.
Many thanks!


![Screenshot_2022-09-19-12-44-46-720_com zwift zwiftgame](https://user-images.githubusercontent.com/100992233/191041682-361a514e-e976-4c60-ac4e-245e69c6c210.jpg)
